starknet_devnet/starknet_wrapper.py

```diff
@@ -469,24 +469,25 @@ def parse_l1_l2_messages(self, l1_raw_messages, l2_raw_messages) -> dict:
         }
 
     def get_state_update(self, block_hash=None, block_number=None):
-        """Returns state update for the provided block hash or the last state update if none provided"""
+        """
+        Returns state update for the provided block hash or block number.
+        It will return the last state update if block is not provided.
+        """
         if block_hash:
```

Comment:
What if both block_hash and block_number are specified? Is the behavior the same as alpha-goerli's?

Reply:
[854fc2d](https://github.com/Shard-Labs/starknet-devnet/pull/49/commits/854fc2dd96d8345b753c35877467c9a7cb30c5cc) it will default to `block_hash`